docs/build/autogenerate.rst

```diff
@@ -405,18 +407,24 @@ is set to True::
 .. note::
 
    The default type comparison logic (which is end-user extensible) currently
```

Comment:
**mike bayer** (zzzeek) wrote:

"as of Alembic version 1.4"   just to make sure people see this